src/com/xilinx/rapidwright/util/PerformanceExplorer.java

```diff
@@ -506,16 +560,48 @@ public void getBestDesignPerPBlock() {
                 System.out.println("No implementation passes timing for " + pblock);
             }
 
+            System.out.println("Best result found at: " + bestPath);
             Design d = Design.readCheckpoint(bestPath + File.separator + "routed.dcp");
 
             if (ensureExternalRoutability()) {
                 InlineFlopTools.removeInlineFlops(d);
+                unrouteNetsThatLeavePBlock(d, getPBlock(pblockNum));
             }
 
+            EDIFTools.ensurePreservedInterfaceVivado(d.getNetlist());
             d.writeCheckpoint(runDirectory + File.separator + pblock + "_best.dcp");
         }
     }
 
+    public void unrouteNetsThatLeavePBlock(Design d, PBlock pBlock) {
```

Comment:
This looks like a handy method and seems like it should be labeled `static`.  Should we refactor it to more generic location, `NetTools` perhaps?